README.md

```diff
@@ -53,6 +53,60 @@
 
     - **session_token**: session token (string, required)
 
+- **client_config**: configure S3 client config (optional)
+
+  - **protocol**: (enum, `HTTP` or `HTTPS`, optional)
+
+  - **max_connections**: (int, optional)
+
+  - **user_agent** (string, optional)
+
+  - **local_address**: name of a hostname (string, optional)
+
+  - **proxy_host**: name of a hostname (string, optional)
+
+  - **proxy_port**: (int, optional)
+
+  - **proxy_username**: (string, optional)
+
+  - **proxy_password**: (string, optional)
+
+  - **proxy_domain**: (string, optional)
+
+  - **proxy_workstation**: (string, optional)
+
+  - **max_error_retry**: (int, optional)
+
+  - **socket_timeout**: (int, optional)
+
+  - **connection_timeout**: (int, optional)
+
+  - **request_timeout**: (int, optional)
+
+  - **use_reaper**: (boolean, optional)
+
+  - **use_gzip**: (boolean, optional)
+
+  - **signer_override**: (string, optional)
+
+  - **preemptive_basic_proxy_auth**: (boolean, optional)
+
+  - **connection_ttl**: (long, optional)
+
+  - **connection_max_idle_millis**: (long, optional)
```

Comment:
embulk usually uses seconds for duration parameters. using milliseconds without _millis suffix for socket_timeout, connection_timeout, etc. sounds confusing. Although AWS SDK itself is the cause of this confusion, I think we should provide better syntax for users.